packages/shared/components/popups/LedgerLegacyTransaction.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+    import { Illustration, Text } from 'shared/components'
+    import { formatUnitBestMatch } from 'shared/lib/units'
+
+    type Output = {
+        address: string
+        value: number
+        checksum: string
+    }
+    type Input = {
```

Comment:
Best to define these types in `shared`. Could possibly be reused [here](https://github.com/iotaledger/firefly/pull/1246/files#diff-502cfa0c2543d63e9efdbfe5989c4b55564c61a34cd305316f94a6216ac5bc0cR146)

Reply:
These types are not exactly shareable, checksum for example is not part of the normal input/output. But I'll give it a thought, thanks!